consensus/src/pipeline/buffer_item.rs

```diff
@@ -95,7 +74,7 @@ fn aggregate_commit_proof(
 // we differentiate buffer items at different stages
 // for better code readability
 pub struct OrderedItem {
-    pub unverified_signatures: PartialSignatures,
+    pub unverified_votes: Vec<CommitVote>,
```

Comment:
using vector here loses the guarantee that one author only has one vote in partial signatures, why this is not a ledger info with unverified signatures?